Xamarin/FormsClient/FormsClient.Android/FormsClient.Android.csproj

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Ugly csproj 😢

Reply:
At least it has `<PackageReference>` and not `packages.config`